src/build/buildNativefierApp.ts

```diff
@@ -109,6 +110,17 @@ export async function buildNativefierApp(
   rawOptions: NativefierOptions,
 ): Promise<string> {
   log.info('Processing options...');
+  if (rawOptions.upgrade && rawOptions.upgrade !== undefined) {
+    const oldApp = findUpgradeApp(rawOptions.upgrade.toString());
+    if (oldApp !== null) {
+      rawOptions = applyOldApp(rawOptions, oldApp);
+    } else {
+      throw Error(
+        `Could not find an old Nativfier app in "${rawOptions.upgrade.toString()}"`,
+      );
+    }
```

Comment:
Nit: when applicable, I like early exits better than if/elses (1. they encourage defensive programming by putting corner cases first, 2. they eliminate an ounce of indentation).

Also, missing `new` before `Error`. I'm surprised ESLint didn't complain about this :thinking: . Can you search why it didn't?

```suggestion
    if (oldApp === null) {
      throw new Error(
        `Could not find an old Nativfier app in "${rawOptions.upgrade.toString()}"`,
      );
    }
    rawOptions = applyOldApp(rawOptions, oldApp);
```